gemrb/core/Scriptable/Actor.cpp

```diff
@@ -7876,7 +7876,10 @@ bool Actor::AdvanceAnimations()
 
 bool Actor::IsDead() const
 {
-	return InternalFlags & IF_STOPATTACK;
+	if (InternalFlags & IF_REALLYDIED) return true;
+	if (Modified[IE_STATE_ID] & STATE_DEAD) return true;
+
+	return false;
```

Comment:
This is also bad without adjusting the users. The internal flags check will catch fewer states and the state check will catch some extra. For some that may be fine, but that's to be proven first. IF_STOPATTACK is clearly there with a specific intention, even if the function name is a bit deceptive.

Reply:
Somewhere the original check did not work for me. I looked at the reference in your code
```
if (ga_flags&GA_NO_DEAD) {
if (InternalFlags&IF_REALLYDIED) return false;
if (Modified[IE_STATE_ID] & STATE_DEAD) return false;
}
```

Reply:
I'm resolving this note, but this change is not going in. If you encounter where you had a faulty check again, document it and we can look at the root problem.